### DISCUSSION TITLE
Combining two schemas merged together

### DISCUSSION THREAD

**Discussion by sgentile** [dotansimha/envelop#846] [Q&A] [answered]
I'm curious if I can use envelop to merge two schemas.

My use case is - I have NextJS and Apollo Server and I have it setup with passing a schema.    This works locally.  Now, I want to add in graphql-mesh.   It has it's getMesh call that returns the schema from the mesh file.

I want to combine my local schema with the getMesh schema and pass to my apollo server.

Is this possible ?

**Comment by saihaj** [comment, marked as answer]
You can use stitch those 2 schemas using [`@graphql-tools/schema`](https://www.graphql-tools.com/docs/schema-stitching/stitch-api) and then pass that schema to your server.
